tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "target": "es2017",
```

Comment:
did these come by default or were these added? Make sure you know what each does if the latter

Reply:
we need outdir to tell it where to put the transpiled JS. I've updated target to ES2020 so we can use newer features